src/main/webapp/js/user-page-loader.js

```diff
@@ -69,31 +68,10 @@ function fetchMessages() {
       });
 }
 
-/**
- * Builds an element that displays the message.
- * @param {Message} message
- * @return {Element}
- */
-function buildMessageDiv(message) {
-  const headerDiv = document.createElement('div');
-  headerDiv.classList.add('message-header');
-  headerDiv.appendChild(document.createTextNode(
-      message.user + ' - ' + new Date(message.timestamp)));
-
-  const bodyDiv = document.createElement('div');
-  bodyDiv.classList.add('message-body');
-  bodyDiv.innerHTML = message.text;
-
-  const messageDiv = document.createElement('div');
-  messageDiv.classList.add('message-div');
-  messageDiv.appendChild(headerDiv);
-  messageDiv.appendChild(bodyDiv);
-
-  return messageDiv;
-}
-
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
+  addFeedLinktoNavigation();
+  addLoginOrLogoutLinkToNavigation();
```

Comment:
Do you think it makes sense to consolidate these two methods?

Reply:
Oops I think I just accidentally closed your other comment but I added the links to the feed page as well but I didn't want to push because I was trying to figure out how to make a separate pull request but that didnt end up working I can consolidate these methods and push them all together if we ignore/decline this pull request but im not quite sure how that works

Reply:
Ah, gotcha! Now that your previous work is merged, the easiest solution might be to make a new branch from master--let me know if there's anything specific you want to debug with the git branches there.

Looks like this PR is already closed, which is totally fine if you want to push a new one, or you can reopen this, merge, and then push more changes. Either way will work!